app/models/invenio_rdm_record_converter.rb

```diff
@@ -79,33 +79,33 @@ def extra_data
     if !@generic_file.based_near.empty?
       data["presentation_location"] = @generic_file.based_near
     end
-    data["permissions"] = file_permissions
 
+    data["permissions"] = file_permissions
     data
   end
 
   def owner_info(depositor)
     user = User.find_by(username: depositor)
 
     if user
-      {user.username => user_email(user)}
+      {user.username => user_email(user.email)}
     else
       {"unknown": "unknown"}
     end
   end
 
   def file_permissions
-    permission_data = Hash.new
-
+    permission_data = {}
```

Comment:
I can't remember exactly why I was using `Hash.new` constructor syntax rather than `{}`, but regardless I think this is better.